test/functional/update_v2/update-client-certificate.bats

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+TEST_DIR_PATH=$(dirname "$(realpath "$TEST_NAME")/$TEST_NAME")
+
+server_pub="$TEST_DIR_PATH/server-pub.pem"
+server_key="$TEST_DIR_PATH/server-key.pem"
+client_pub="$TEST_DIR_PATH/client-pub.pem"
+client_key="$TEST_DIR_PATH/client-key.pem"
+
+global_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_test_environment "$TEST_NAME" 100
+	
```

Comment:
This a very minor thing, but avoid leaving trailing spaces and tabs in code. This is a bit annoying in some editors or when using git diff with colors.

Reply:
I'll clean this up.